### ISSUE TITLE
User Full Name is incorrect

### ISSUE THREAD

**Issue by hjmallon** [cockpit-project/cockpit#11459]
Cockpit version: 176
OS: CentOS 7.6
Page: Accounts

User Full Name is read from GECOS field. However GECOS field has it's own substructure (comma separated with a few different fields, see command `chfn`). So it should read out the first part before a comma of the GECOS field and write back the new name followed by the rest of the field (or offer changing of all GECOS features).

Steps to reproduce:
```
`chfn $user`
# Follow chfn prompts
# Check /etc/passwd for what happened
```

**Comment by KKoukiou** [comment]
@hjmallon thanks for reporting this! I have some concerns however.

It's true that in case you 're using the other finger fields, (office, phone etc) they will appear in the full name field. Since they are not really part of the users full name, this might look weird from first sight. But taking away the naming of the label, there is nothing broken there. You can still edit the fields and the full name from that field.

I doubt that we want to introduce new fields for editing/showing all GECOS features, honestly I never saw anyone using them.

Your suggestion, to parse the users full name to not show anything after first comma will remove the option to edit these extra fields (which is possible now, although now done indirectly through the name)

So I suggest to keep this as is, having the `Full Name` label be the whole comment section of a user in /etc/passwd, since the only confusing part is the label name and it's indirectly offering the editing functionality for these fields.

**Comment by martinpitt** [comment]
I agree that we can leave the full field in the "Full Name" text input, but I think we should only show the actual name in the user list view and the per-user table header.

Incidentally, [chfn is missing on Fedora 29](https://bugzilla.redhat.com/show_bug.cgi?id=1693955).

Marking as "good-first-issue", this is an excellent finger exercise for new contributors.

**Comment by martinpitt** [comment]
Fixed in PR #11494
